### PR TITLE
Disambiguate defining and installing Python packages from dbt packages

### LINKED ISSUE #1869: Further disambiguate dbt packages (and `packages.yml`) from Python packages, configured per-model
See this feedback: https://dbt-labs.slack.com/archives/C03C17F0PC6/p1660668992786679

We can add a note or something in the Python models docs to explain

### PR DESCRIPTION
**Pull request by dbeatty10**
resolves #1869

## What are you changing in this pull request and why?

We heard at least one piece of feedback on the term packages:
- While [the docs](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models) do well to call out Python packages, there might be a need for further disambiguation between Python packages and dbt packages.
- The question was if Python packages needed to be defined in the `packages.yml` file.

This PR attempts to make a brief callout that provides a disambiguating link.

Happy to incorporate any feedback if these aren't the docs we want or if we don't feel we even need this disambiguation anymore.

## Preview Link

https://deploy-preview-2862--docs-getdbt-com.netlify.app/docs/build/packages#what-is-a-package

## Screenshots 🎩

image

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/build/packages.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ dbt _packages_ are in fact standalone dbt projects, with models and macros that
 * You can use `ref` in your own models to refer to models from the package.
 * You can use macros in the package in your own project.
 
+:::note
+
+Defining and installing dbt packages is different than defining and installing [Python packages](https://docs.getdbt.com/docs/build/python-models#using-pypi-packages).
```
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]: committing relative url paths

```suggestion
Defining and installing dbt packages is different than defining and installing [Python packages](/docs/build/python-models#using-pypi-packages).
```

File: website/docs/docs/build/packages.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ dbt _packages_ are in fact standalone dbt projects, with models and macros that
 * You can use `ref` in your own models to refer to models from the package.
 * You can use macros in the package in your own project.
 
+:::note
```
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]:
```suggestion
:::note Using Python packages
```

File: website/docs/docs/build/packages.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ dbt _packages_ are in fact standalone dbt projects, with models and macros that
 * You can use `ref` in your own models to refer to models from the package.
 * You can use macros in the package in your own project.
 
+:::note Using Python packages
+
+Defining and installing dbt packages is different than defining and installing [Python packages](/docs/build/python-models#using-pypi-packages).
```
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]: hyperlinking that action so users are absolutely clear the doc is about defining and installing pypi packages, not just about python.

```suggestion
Defining and installing dbt packages is different from [defining and installing Python packages](/docs/build/python-models#using-pypi-packages).
```

**Review by mirnawong1 — APPROVED** [review]
hey @dbeatty10 great callout and thank you very much for providing that info! I've taken the liberty to make some small relative URL tweaks so speed up the process for you, however, please do let me know if you disagree with any of the suggestions!

it's approved for you to merge ✨

**Comment by dbeatty10** [comment]
Thank for for speeding up this process by committing those improvements @mirnawong1 ! 🙌

**Pull request merged**